FIAT/mardal_tai_winther.py

```diff
@@ -0,0 +1,172 @@
+# -*- coding: utf-8 -*-
+"""Implementation of the Mardal-Tai-Winther finite elements."""
+
+# Copyright (C) 2019 Robert Kirby <robert.c.kirby@gmail.com>
```

Comment:
This is perhaps not copy-paste.

Reply:
I am no lawyer, but is copyright not normally held by the employer for those works which are created for work purposes or "during work hours"?

Reply:
Not sure how this works in academia, but given the age of the project and lack of complaints, I will proceed as before.